### ISSUE TITLE
Broken internal import with release `1.0.3`

### ISSUE THREAD

**Issue by iolyd** [hosseinmd/prettier-plugin-jsdoc#209]
# Bug description

After updating to the latest version (`1.0.3`), attempting to format a document with prettier yields:

> Cannot find module '/[omitted]/node_modules/.pnpm/prettier-plugin-jsdoc@1.0.3_prettier@3.0.3/node_modules/prettier-plugin-jsdoc/dist/parser.js'
imported from /[omitted]/node_modules/.pnpm/prettier-plugin-jsdoc@1.0.3_prettier@3.0.3/node_modules/prettier-plugin-jsdoc/dist/index.js

Reverting back to `1.0.2` fixes this error.

## Additionnal info

This is happening in a SvelteKit project scaffolded with a pnpm repo.

**Comment by panva** [comment]
Same here, not on all repos tho

reproduce with

```
gh repo clone panva/jose
npm ci
npm install prettier-plugin-jsdoc@1.0.3
npm run format
```

**Comment by hosseinmd** [comment]
fixed: v1.0.4
